autopush-common/src/reliability.rs

```diff
@@ -118,7 +125,11 @@ impl PushReliability {
                     .unwrap_or_else(|| "None".to_owned()),
                 new
             );
-            match client.get_connection_with_timeout(CONNECTION_EXPIRATION) {
+            match client.get_connection_with_timeout(
+                CONNECTION_EXPIRATION
+                    .to_std()
+                    .expect("Could not convert const CONNECTION_EXPIRATION."),
```

Comment:
I think the usage of `map_err` vs `expect` should be consistent in the 3 places in this module. However, more importantly, w/ `to_std` returning a `Result`, I'd argue this is one of those cases where the usage is specifically limited to `get_connection_with_timeout`'s argument that we should just keep the original `std::time::Duration` to avoid `to_std`'s noise (sigh).